internal/app/donor_store.go

```diff
@@ -170,25 +179,35 @@ func (s *donorStore) Put(ctx context.Context, lpa *page.Lpa) error {
 		}
 	}
 
-	if lpa.UID != "" && lpa.Tasks.PayForLpa.IsPending() && lpa.HasUnsentReducedFeesEvidence() {
+	if lpa.UID != "" && lpa.Tasks.PayForLpa.IsPending() {
+		documents, err := s.documentStore.GetAll(ctx)
+		if err != nil {
+			return err
+		}
+
 		var unsentKeys []string
 
-		for _, document := range lpa.Evidence.Documents {
-			if document.Sent.IsZero() {
+		for _, document := range documents {
+			if document.Sent.IsZero() && !document.Scanned {
 				unsentKeys = append(unsentKeys, document.Key)
 			}
 		}
 
-		if err := s.eventClient.Send(ctx, "reduced-fee-requested", reducedFeeRequestedEvent{
-			UID:         lpa.UID,
-			RequestType: lpa.FeeType.String(),
-			Evidence:    unsentKeys,
-		}); err != nil {
-			s.logger.Print(err)
-		} else {
-			for i, document := range lpa.Evidence.Documents {
-				if document.Sent.IsZero() {
-					lpa.Evidence.Documents[i].Sent = s.now()
+		if len(unsentKeys) > 0 {
+			if err := s.eventClient.Send(ctx, "reduced-fee-requested", reducedFeeRequestedEvent{
+				UID:         lpa.UID,
+				RequestType: lpa.FeeType.String(),
+				Evidence:    unsentKeys,
+			}); err != nil {
+				s.logger.Print(err)
+			} else {
+				for _, document := range documents {
+					if document.Sent.IsZero() && !document.Scanned {
+						document.Sent = s.now()
+						if err := s.documentStore.Put(ctx, document, nil); err != nil {
+							return err
```

Comment:
We shouldn't be returning errors from here. It should just fail, log, and retry next time

Reply:
also would be nice to batch these changes to documents up